lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +14,34 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      return nil unless ENV.key?('EXPORT_DATE_OVERRIDE')
+
+      today = Date.today
+      days_ago = (0..30).find do | days_ago |
+        exists_export? date: today-days_ago
+      end
+      
+      if days_ago.nil? || days_ago == 0
+        today.strftime( DATE_FORMAT )
+      else
+        ( today - days_ago.to_i + 1).strftime( DATE_FORMAT )
+      end
```

Comment:
the conditional here around `day_ago`, but the result is `date.strftime`. Is there a way to consolidate the `days_ago` logic into a separate statement to have the final lines of the function be
```ruby
  days_ago = # maths
  (today - days_ago).strftime( DATE_FORMAT )
```

Reply:
Good catch. I introduced some bug here.